source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -1923,6 +1923,8 @@ oversaw the specification process.
 
 - 2024-11-04: Make the description of `TopologyDescription.servers` consistent with the spec tests.
 
+- 2024-10-29: Removed references to `getLastError`
```

Comment:
Bookmark to make sure to update the changelog to the current date once all other comments are addressed